src/main/java/com/robosh/ejournal/entity/Group.java

```diff
@@ -27,9 +30,14 @@
 
     @NotBlank
     @Pattern(regexp = "\\d{1,2}[-]+[А-ЯЇ]")
+    @Column(name = "code")
     private String code;
 
+
     @OneToMany
+    @JoinTable(name = "student_group",
+            joinColumns = @JoinColumn(name = "group_id"),
+            inverseJoinColumns = @JoinColumn(name = "student_list"))
     private List<Student> studentList;
```

Comment:
It's better to name student_id, as it's not a list, it's only one cell.

Reply:
my bad